c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -1377,6 +1381,22 @@ bool body_reader_then_handler(JSContext *cx, HandleObject body_owner, HandleValu
     fprintf(stderr, "Error: read operation on body ReadableStream didn't respond with a "
```

Comment:
I guess you should edit this comment while you're at it.

Reply:
I changed the comment to reflect the open question about handling Responses that have failed to read a `Uint8Array`, and moved the error printing to below the promise rejection for a failed request.